src/main/resources/com/databricks/spark/sql/perf/mllib/config/mllib-large.yaml

```diff
@@ -37,6 +37,28 @@ benchmarks:
       numFeatures: 5000
       numClasses: 2
       smoothing: 1.0
+  - name: clustering.GaussianMixture
+    params:
+      numExamples: 100000
+      numTestExamples: 100000
+      numFeatures: 1000
+      k: 10
+      maxIter: 10
+      tol: 0.01
+  - name: clustering.KMeans
```

Comment:
what is the dataset size for kmeans?

Reply:
The size is 1000000.

Reply:
But why it is not a param?

Reply:
We have some common default parameters:
common:
  numExamples: 1000000
  numTestExamples: 1000000
  numFeatures: 4000
  numPartitions: 64
  randomSeed: [1, 1, 1]

These will be the params for the benchmark call, but we don't need to write it for each case.